Formula/jdtls.rb

```diff
@@ -0,0 +1,63 @@
+class Jdtls < Formula
+  desc "Java language specific implementation of the Language Server Protocol"
+  homepage "https://github.com/eclipse/eclipse.jdt.ls"
+  url "https://download.eclipse.org/jdtls/milestones/1.7.0/jdt-language-server-1.7.0-202112161541.tar.gz"
+  sha256 "2f0c28dfec317a268ec44904420657181b43a7ba2a32f0bf788ea388dacb8552"
+  license "EPL-2.0"
+
+  depends_on "openjdk@11"
+
+  def install
+    etc.install "config_mac" => "jdtls"
+    libexec.install "features"
+    libexec.install "plugins"
+
+    (buildpath/"jdtls-wrapper").write <<~BASH
+      #!/bin/bash
+      # copy the configuration folder to tmp to be writable
+      tmp_dir="$(mktemp -d /tmp/jdtls.XXXXX)"
+      cp -R #{etc}/jdtls "${tmp_dir}"
+      # and ensure that it is removed on exit
+      trap "{ rm -rf ${tmp_dir}; }" EXIT
+
+      java -Declipse.application=org.eclipse.jdt.ls.core.id1 \\
+        -Dosgi.bundles.defaultStartLevel=4 \\
+        -Declipse.product=org.eclipse.jdt.ls.core.product \\
+        -jar #{libexec}/plugins/org.eclipse.equinox.launcher_*.jar \\
+        --add-modules=ALL-SYSTEM \\
+        --add-opens java.base/java.util=ALL-UNNAMED \\
+        --add-opens java.base/java.lang=ALL-UNNAMED \\
+        -configuration "${tmp_dir}/jdtls" \\
+        "$@"
```

Comment:
Are all these options required to make the program work in the first place? Why doesn't upstream ship a script that sets this because this seems very fragile, again: https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-heavy-manual-prepost-install-intervention

Reply:
I'll look into this again. Maybe I'll find a better solution.